ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/io/TableSerializer.java

```diff
@@ -96,6 +98,11 @@ private Table updatePropertiesInTable(Table table, ReplicationSpec additionalPro
       // uncomment this else section, but currently unneeded. Will require a lot of golden file
       // regen if we do so.
     }
+
+    if (parameters != null && parameters.containsKey(COLUMN_STATS_ACCURATE)) {
```

Comment:
There is already parameters != null check above in this method. Can we move this code there itself?